components/wmtk_components/input/input.cpp

```diff
@@ -52,11 +56,65 @@ void input(const nlohmann::json& j, std::map<std::string, std::filesystem::path>
         files[options.name] = cached_mesh_file;
         break;
     }
-    case 1:
+    case 1: {
         // edge mesh
-        throw "not implemented";
-        assert(false);
+        spdlog::warn("this edge mesh is old version, need to update according to the latest "
```

Comment:
Now as you have the `EdgeMesh` available, you should remove the warning and adapt the code.